docs/user/tutorials/01-00-x-forwarded-for-header.md

```diff
@@ -0,0 +1,103 @@
+# Forward Client IP in X-Forwarded-For header
+
+Many applications need to know the client IP address of an originating request to behave properly. Usual use-cases include workloads that require the 
+client IP address to restrict their access. The ability to provide client attributes to services has long been a staple of reverse proxies. 
+To forward client attributes to destination workloads, proxies use the X-Forwarded-For (XFF) header. For more information on XFF, see 
+the [IETF’s RFC documentation](https://datatracker.ietf.org/doc/html/rfc7239) and [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-for).
+
+> **NOTE:** The X-Forwarded-For header is only supported on AWS clusters.
+
+## Prerequisites
+
+* An AWS cluster
+* The Istio module enabled or [Kyma Istio Operator](../../../README.md#install-kyma-istio-operator-and-istio-from-the-latest-release) installed
+* The API Gateway module enabled or [Kyma API Gateway Operator](https://github.com/kyma-project/api-gateway#installation) installed. Alternatively, you can use [custom Istio Gateway](https://kyma-project.io/#/api-gateway/user/tutorials/01-20-set-up-tls-gateway).
```

Comment:
is it necessary to mix in api-gateway module?

Reply:
This mentions both approaches, I think it should stay like this or maybe just change the order.